Objects/codeobject.c

```diff
@@ -11,9 +11,23 @@
 #include "pycore_tuple.h"         // _PyTuple_ITEMS()
 #include "clinic/codeobject.c.h"
 
+static PyObject* code_repr(PyCodeObject *co);
+
+static const char *
+code_event_name(PyCodeEvent event) {
+    switch (event) {
+        #define CASE(op)                \
+        case PY_CODE_EVENT_##op:         \
+            return "PY_CODE_EVENT_" #op;
+        FOREACH_CODE_EVENT(CASE)
+        #undef CASE
+    }
```

Comment:
Need a default and/or error return after the switch.

Reply:
Ugh, thanks, been writing too much C++ with non-exhaustive switch errors turned on.